REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.3 0.1
+ForwardDiff # Currently only required for testing.
```

Comment:
>
> That doesn't look so bad, if we're just using ForwardDiff.jl for testing
> rather than GSL.jl. Or did you have in mind using ForwardDiff.jl outside of
> the test directory?
>

I admit that when I saw the similarity between some of what they've done
and some of what I've started to do for the Hessian, I thought that my
procrastination had really paid off.  But evidently not.